README.md

```diff
@@ -225,8 +225,8 @@ end
 Route patterns may have optional parameters:
 
 ``` ruby
-get '/posts.?:format?' do
-  # matches "GET /posts" and any extension "GET /posts.json", "GET /posts.xml" etc.
+get '/post/:format?' do
+  # matches "GET /posts" and any extension "GET /posts/json", "GET /posts/xml" etc
```

Comment:
`get /posts/:year?` would be ok but the the `/?` means the problem would remain, as it'd match posts_2005 as a `:year` of `_2005`